package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@swapr/periphery",
-  "version": "0.3.20",
```

Comment:
Shouldn't this be a minor update?

Reply:
I'd consider it a minor update if changes on the code/build process were done IMHO, historically, I think I've always treated address updates as patches, since simply the `.contracts.json` file has been updated.

Reply:
Got it. Let's merge this then!